CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+v0.2.0
```

Comment:
I copied the section headers from Node-RED's CHANGELOG.md so all the other projects start with

```
#### 0.2.0: Release
```